go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/go-vela/pkg-executor
 go 1.15
 
 require (
-	github.com/gin-gonic/gin v1.7.1
+	github.com/gin-gonic/gin v1.7.2
 	github.com/go-vela/compiler v0.7.4
 	github.com/go-vela/mock v0.7.5-0.20210416160452-fa8721fcc6a1
-	github.com/go-vela/pkg-runtime v0.7.5-0.20210428154035-f007d6e59a10
+	github.com/go-vela/pkg-runtime v0.7.5
```

Comment:
`v0.7.5` was a backport fix:

https://github.com/go-vela/pkg-runtime/pull/115

Merging this would be a downgrade in functionality so we should block this.